### PR TITLE
Optimize java doc in ShardingRule class

### PR DESCRIPTION
**Pull request by KSMASTERS**
Fixes #10461.

Changes proposed in this pull request:
- Optimize java doc in ShardingRule class
-
-

### CODE REVIEW

**Comment by tristaZero** [comment]
@KSMASTERS Hey, thanks for your contribution.

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java [hunk 1]
```diff
@@ -240,10 +240,10 @@ public TableRule getTableRule(final String logicTableName) {
     }
     
     /**
-     * Judge logic tables is all belong to binding encryptors.
+     * Judge whether all logic tables belongs to binding encryptors.
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-common/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java:243]: @KSMASTERS What do you think of this? 

```java
/**
 * Judge whether all logic tables are binding encryptors or not.
 * 
 * @param logicTableNames logic table names
 * @return whether all logic tables are binding encryptors or not
 */
```

If you think it's good, please use similar expressions to modify the Java doc.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Comment by KSMASTERS** [comment]
Can I just submit it directly?

**Comment by tristaZero** [comment]
@KSMASTERS You mean PR? Sure, welcome.

**Pull request closed without merging**